pkg/multiTenancyPlugins/utils/utils.go

```diff
@@ -300,14 +300,11 @@ func commandParser(r *http.Request) string {
 			strings.Contains(r.RequestURI, "/volumes?") {
 			return "volumeslist"
 		}
-		if len(volumesParams) == 4 && volumesParams[3] != "" {
+		if len(volumesParams) == 4 {
 			if volumesParams[3] == "create" {
 				return "volumecreate"
 			}
 			return "volumeinspect"
-		} else {
-			log.Debugf("I am here", volumesParams)
-			return "volumeinspect"
```

Comment:
This is deleted code
-		} else {
-			log.Debugf("I am here", volumesParams)
-			return "volumeinspect"